app/views/candidates/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="vertical-tabs-container">
+  <div class="vertical-tabs">
+    <% @candidates.each_with_index do |candidate, index| %>
+      <a href="javascript:void(0)" class="js-vertical-tab vertical-tab <%= 'is-active' if index==0 %>" rel="tab<%= candidate.id%>"><%= candidate.name %></a>
+    <% end %>
+  </div>
+
+  <div class="vertical-tab-content-container">
+    <% @candidates.each do |candidate| %>
+      <a href="" class="js-vertical-tab-accordian-heading vertical-tab-accordion-heading" rel="tab<%= candidate.id %>"><%= candidate.name %></a>
+      <div id="tab<%= candidate.id %>" class="js-vertical-tab-content vertical-tab-content">
+        <div class="grid-items">
+          <div class="grid-item">
+            <%= image_tag candidate.photo_url, size: "100x100" %>
+          </div>
+          <div class="grid-item grid-item-big">
+            <% unless candidate.twitter.nil? %>
```

Comment:
What do you think about changing this to a positive check? `if candidate.twitter.present?`